plugins/modules/pnp_intent.py

```diff
@@ -838,23 +874,30 @@ class instance for further use.
             op_modifies=True,
             params=provisioned_count_params,
         )
+        self.log("Devices which are provisioned response from the API 'get device count' is {0}".format(str(prov_dev_response)), "DEBUG")
```

Comment:
self.log("Response from 'get device count' API for provisioned devices: {0}".format(str(prov_dev_response)), "DEBUG")

Reply:
Resolved